src/geocat/f2py/missing_values.py

```diff
@@ -1,42 +1,52 @@
 import numpy as np
 
-#all missing values are represented by all 1's in their dtype, int8 would be bin_11111111 or dec_-128
+# all missing values are represented by the maximum value in their dtype,
```

Comment:
Is it intentional to change the use of minimum to maximum? I'd expect using min to be better safe than max (requiring the sign-bit to be 1 in addition to the rest of the bits) for missing value representation.